app/components/Poll/Poll.css

```diff
@@ -1,16 +1,83 @@
 @import url('~app/styles/variables.css');
 
 .poll {
-  min-height: 160px;
+  background-color: var(--lego-card-color);
+  width: 100%;
+  min-width: 250px;
+  max-width: 500px;
+  min-height: 120px;
+  margin: 0 auto;
+  overflow-y: hidden;
+  border-radius: 5px;
+}
+
+.topBar {
+  width: 100%;
+  z-index: 2;
+}
+
+.pollIcon {
+  display: block;
+  width: 100%;
+  text-align: center;
+  background-color: var(--lego-red-color);
+  color: var(--lego-card-color);
```

Comment:
hmm are you sure you want to use the card background color for as a "text" color (I can see it's an icon)?

Reply:
Whop - nopes, changed it to le white white now